server/model/reviewsModel.js

```diff
@@ -0,0 +1,73 @@
+import pool from '../../database/db.js';
+
+export const listReviews = (reqQuery, callback) => {
+  // need to add query to add photos and allow for sorting
+  const {
+    page = 1, count = 5, sort, product_id,
+  } = reqQuery;
+
+  const query = 'SELECT review_id, rating, summary, recommend, response, body, date, reviewer_name, helpfulness FROM reviews WHERE product_id = $1 LIMIT $2;';
+
+  pool.query(query, [product_id, count], (error, results) => {
+    if (error) {
+      callback(error, null);
+    } else {
+      const rows = {
+        product: product_id,
+        page,
+        count,
+        results: [
+          ...results.rows,
+        ],
+      };
+      callback(null, rows);
+    }
+  });
+};
+
+export const addReview = (data, callback) => {
+  const {
+    product_id, rating, summary, body, recommend,
+    name, email, photos, characteristics,
+  } = data;
+
+  const date = new Date();
+
+  const insertIntoReviews = 'INSERT INTO reviews(product_id, rating, summary, body, recommend, reviewer_name, reviewer_email, date, reported, helpfulness) values($1, $2, $3, $4, $5, $6, $7, $8, $9, $10) RETURNING review_id;';
```

Comment:
These query variables are super clean. Minor, but the more informational name here (insertIntoReviews) versus "query" on the others is nice.